### PR TITLE
Add numerical representation of Consul health check state.

### PR DESCRIPTION
**Pull request by johnrengelman**
### Required for all PRs:

- [x] CHANGELOG.md updated (we recommend not updating this until the PR has been approved by a maintainer)
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [x] README.md updated (if adding a new plugin)

### CODE REVIEW

**Comment by johnrengelman** [comment]
I targeted this to `master`, but it would be nice to be in `1.2` since that is still unreleased.

**Comment by sparrc** [comment]
this looks good, but I think these should be integers rather than floats

**Comment by johnrengelman** [comment]
I thought I'd have to do `= 1.0` to get a float?

**Comment by sparrc** [comment]
I would have thought so too, but this line-protocol shows they are floats: `passing=1,critical=0,warning=0` (there would be an `i` appended to each number if they were integers).

**Comment by johnrengelman** [comment]
oh ha. Yeah, I just manually typed that. I'll verify and update according. :)

**Comment by johnrengelman** [comment]
Updated.

File: plugins/inputs/consul/README.md [hunk 1]
```diff
@@ -35,12 +35,19 @@ Fields:
...
 
 ## Example output
 
 ```
 $ telegraf --config ./telegraf.conf -input-filter consul -test
 * Plugin: consul, Collection 1
-> consul_health_checks,host=wolfpit,node=consul-server-node,check_id="serfHealth" check_name="Serf Health Status",service_id="",status="passing" 1464698464486439902
-> consul_health_checks,host=wolfpit,node=consul-server-node,service_name=www.example.com,check_id="service:www-example-com.test01" check_name="Service 'www.example.com' check",service_id="www-example-com.test01",status="critical" 1464698464486519036
+> consul_health_checks,host=wolfpit,node=consul-server-node,check_id="serfHealth" check_name="Serf Health Status",service_id="",status="passing",passing=1i,critical=0i,warning=0i 1464698464486439902
+> consul_health_checks,host=wolfpit,node=consul-server-node,service_name=www.example.com,check_id="service:www-example-com.test01" check_name="Service 'www.example.com' check",service_id="www-example-com.test01",status="critical",passing=0i,critical=1i,warning-0i 1464698464486519036
```
Comment (sparrc) [inline plugins/inputs/consul/README.md]: sp: `warning-0i`, still manually typing? ;-)

Comment (johnrengelman) [inline plugins/inputs/consul/README.md]: You caught me. 😈 
Fixed.

**Comment by sparrc** [comment]
:+1:, just update the changelog and I'll merge

**Comment by johnrengelman** [comment]
Update the changelog and rebased.

**Pull request merged**